x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/aws_credentials_form/aws_credentials_form_agentless.tsx

```diff
@@ -36,106 +36,6 @@ import {
   AwsCredentialTypeSelector,
 } from './aws_credentials_form';
 
-const CLOUD_FORMATION_EXTERNAL_DOC_URL =
-  'https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-whatis-howdoesitwork.html';
-
-export const CloudFormationCloudCredentialsGuide = ({
-  isOrganization,
-}: {
-  isOrganization?: boolean;
-}) => {
-  return (
-    <EuiText>
-      <p>
-        <FormattedMessage
-          id="xpack.csp.agentlessForm.cloudFormation.guide.description"
-          defaultMessage="CloudFormation will create all the necessary resources to evaluate the security posture of your AWS environment. {learnMore}."
-          values={{
-            learnMore: (
-              <EuiLink
-                href={CLOUD_FORMATION_EXTERNAL_DOC_URL}
-                target="_blank"
```

Comment:
@maxcold  I'm actually use this code in next [PR](https://github.com/elastic/kibana/pull/189932)

Reply:
does it make sense to bring it back as a part of the rebase? either in this PR or yours PR whatever is merged first

Reply:
we discussed in DM and we will deal it with the rebase